pr_line_related_po_line/models/purchase_requisition_line.py

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
I'm guessing the way you removed extra headers left blank lines at the top.

The same applies for the other model.